srcpkgs/python3-stem/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'python3-stem'
+pkgname=python3-stem
+version=1.8.0
+revision=1
+archs=noarch
+wrksrc="stem-${version}"
+build_style=python3-module
+pycompile_module="stem"
```

Comment:
This one should be auto detected by the hook

Reply:
Yep, I think this was done possibly before I found out about not adding and start removing pycompile line.